docs/documentation/stories/disk-serve.md

```diff
@@ -0,0 +1,23 @@
+# Serving from Disk
+
+The CLI supports running a live reload experience to users by running `ng serve`. This will compile the application upon file saves and reload the browser with the newly compiled application. This is done by hosting the application in memory and serving it via [webpack-dev-server](https://webpack.js.org/guides/development/#webpack-dev-server).
+
+If you wish to get a similar experience with the application output to disk please use the steps below.
```

Comment:
My motivation for this is to verify that what will be in the `/dist` folder , which is also what I may send to a production server later, is indeed ok when served. This is important for folks who want to be super careful. I personally dont trust "well, it ran in memory, so it will certainly run off disk" ... why? because maybe somewhere in the black box there is something different. 

Point here is that I think we should add something mentioning this motivation. And that it is easy to do your own verification following your forthcoming steps.